src/models/Offer.js

```diff
@@ -110,9 +110,28 @@ OfferSchema.query.current = function() {
     });
 };
 
+OfferSchema.query.activeOffersCount = function(canditateCompany) {
```

Comment:
Rename this to `activeOffers`, as it returns the offers, not only a number (count)

Reply:
Also, this should be in the Company Model, I think, as you want the active offers of a specific Company, it doesn't make sense to query all offers to check which ones have the specified company as an owner, and then check if they are active. What you want to do is move this to the Company model, and then `Company.find({_id: req.body.owner}).activeOffers` in the validator